### ISSUE TITLE
Line wrap occurring for non-latin scripts

### ISSUE THREAD

**Issue by gorjious** [typemytype/drawbot#254]
Hello,

I would like to report a bug which causes line wraps when there are no width constraints. This seems to be a script related issues because I only encounter it with non-latin scripts (i.e. Armenian, Cyrillic). 

I've attached a drawbot script and font file which illustrates this issue. 

Note: The font file is simplified variable font with three scripts ( Latin, Armenian, and Cyrillic). For simplicity, there is only one letter per script ('N', 'Ո', 'П' and each letter uses the same rectangle with varying widths across 3 masters. 

<img width="1280" alt="screenshot 2019-01-11 at 14 23 10" src="https://user-images.githubusercontent.com/23149420/51060289-874c6600-15ac-11e9-9c39-22ba6137f25b.png">

[drawbot_script_test.zip](https://github.com/typemytype/drawbot/files/2751173/drawbot_script_test.zip)

**Comment by typemytype** [comment]
there is a mismatch between the size retrieved from the attributed string and the size while typesetting the string.

https://github.com/typemytype/drawbot/blob/master/drawBot/drawBotDrawingTools.py#L1598

I guess drawBot needs to get the size of the given text in a different way...

**Comment by gorjious** [comment]
ah okay. thanks for taking a look at it. hopefully it's an easy fix. in the meantime, I found a quick hack to avoid wrapping by adding a Zero-Width space ( \u200b ) at the end.

**Comment by typemytype** [comment]
see https://github.com/typemytype/drawbot/pull/255

**Comment by justvanrossum** [comment]
The fix causes #265 and must be reverted.

**Comment by frankrolf** [comment]
I also noticed this now:
```
font("Minion3-Regular")
fontSize(50)
language('bg')
text("абвгдежзийклмнопрстуфхцчшщъыьэюя", (20, 100))

```

There must be a fixed w/o regressing back to #265 :-)

**Comment by typemytype** [comment]
Dont have Minion3... what is the (un)expected result?

**Comment by typemytype** [comment]
when `bg` is not active it works like it should be... 

so when the locl fea is active and a substitution has happend.

**Comment by frankrolf** [comment]
I don’t know if the `locl` has much to do with it. Even if I remove the language, some fonts wrap:
<img width="1792" alt="Screen Shot 2019-04-09 at 17 01 40" src="https://user-images.githubusercontent.com/2049645/55811282-2f5cc700-5ae9-11e9-9c08-c9d861088c08.png">

**Comment by typemytype** [comment]
this issues both the `ՈՈՈПППNNN` and the Bulgarian replacement should be solved in #319 

a release will follow!
